solver/src/http_solver/settlement.rs

```diff
@@ -8,22 +8,29 @@ use model::order::OrderKind;
 use primitive_types::{H160, U256};
 use std::collections::HashMap;
 
+// To send an instance to the solver we need to identify tokens and orders through strings. This
+// struct combines the created model and a mapping of those identifiers to their original value.
+pub struct PreparedModel {
```

Comment:
nit: Does `model` need to be a part of this? It doesn't seem required for `convert_settlement`. Maybe `prepare_model` can return a pair of `BatchAuctionModel` and `SettlementContext`, the latter consisting only of the three mappings.